packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.0]
+### Removed
```

Comment:
The linter wants this category to come after "Changed". The tool we have is quite strict (not for good reason, it was just easier to build that way)

Reply:
Fixed in https://github.com/MetaMask/core/pull/1708/commits/84faabcee1003040fbb6393b513aa9f32ab002bd